lib/web/sessions.go

```diff
@@ -856,9 +856,9 @@ func (s *sessionCache) invalidateSession(ctx context.Context, sctx *SessionConte
 	if err != nil && !trace.IsNotFound(err) {
 		return trace.Wrap(err)
 	}
-	if err := clt.DeleteUserAppSessions(ctx, &proto.DeleteUserAppSessionsRequest{Username: sctx.GetUser()}); err != nil {
-		return trace.Wrap(err)
-	}
+	// if err := clt.DeleteUserAppSessions(ctx, &proto.DeleteUserAppSessionsRequest{Username: sctx.GetUser()}); err != nil {
+	// 	return trace.Wrap(err)
+	// }
```

Comment:
we can differentiate between "logout" from "idle" and "explicit"

for explicit logout, we can invalidate both web and app sessions

for idle, we'll let web and app handle it separately